README.md

```diff
@@ -146,11 +146,11 @@ You can also do this manually by:
 In order to use the new architecture some extra steps are needed. 
 #### iOS 
 - Install pods with new arch flag inside `example/ios` folder: `RCT_NEW_ARCH_ENABLED=1 pod install`
-- Run `npm run example-ios`
+- Run `npm run ios` in `example/ios` folder
 
 #### Android 
 - Set `newArchEnabled` to true inside `example/android/gradle.properties`
-- Run `npm run example-android`
+- Run `npm run example-android` _(first run might fail, try to run it again)_
```

Comment:
This is worrying... What do we mean here by "first run"? Do we mean the build? Or the app launch?
We don't have anything like that reported and I didn't observe it yet neither in my work with example app, neither in CI builds.

Let me double check it so we make 100% sure it's not a local issue of yours that we put to official docs - for that can you create also the issue in repo so I can check it in repro app?

Reply:
This happened in this scenario:
- clone project
- run install
- run android on the old arch
- change settings to the new arch
- run android on the new arch (without cleaning android <-- might be the problem)

It might have failed on some of the downloads that were going on, but not sure. If you prefer, I can remove the note in the README. Just put it there so people might try it out once more before creating an issue

Reply:
@slavikdenis Thanks for explaining 👍 
Hm, so if this issue appears without cleaning, then I'd prefer to have a note in README that the clean of Android build is suggested when building new arch on an old arch.
Yes, let's remove that from this PR - I think that no matter what kind of note we decide to put in README it's always better to separate such changes between two PRs.
But thank you for noticing this and letting me know, it's definitely something worth revisiting.

Reply:
Removed via https://github.com/callstack/react-native-slider/pull/449/commits/8303f8af7a29b4ba8b48d85d622c4a91f6f86e8b

Let's see if it occurs to anyone else 👍